src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -1125,4 +1129,16 @@ function (Document $function) use ($deleteByFunction) {
             );
         }
     }
+
+    /**
+     * @throws Exception
+     * @throws Throwable
+     */
+    public function deleteTeams($dbForConsole, $getProjectDB, $team, $project): void
```

Comment:
Please add typing for these params. Also, this should be singular (`deleteTeam` instead of `deleteTeams`), right?

Reply:
The method can delete more than one team